app/common/services/maps.js

```diff
@@ -55,22 +55,51 @@ function (
     var geojsonLayerOptions = {
         onEachFeature: function (feature, layer) {
             var description = feature.properties.description || '',
-                title = feature.properties.title || feature.properties.id;
+                title = feature.properties.title || feature.properties.id,
```

Comment:
I think we should try to use angular $compile and render the `post-card` directive here.. otherwise we're repeating the same code in 2 places. There's very little difference between a post card in the list view, and a post card in the popup.